xml/System/Type.xml

```diff
@@ -13196,8 +13211,8 @@ Types that are defined in the .NET Standard are not marked with <xref:System.Ser
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value that indicates whether the <see cref="T:System.Type" /> is an array type that can represent only a single-dimensional array with a zero lower bound.</summary>
+        <value><see langword="true" /> if the current <see langword="Type" /> is an array type that can represent only a single-dimensional array with a zero lower bound; otherwise, <see langword="false" />.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the current <see langword="Type" /> is an array type that can represent only a single-dimensional array with a zero lower bound; otherwise, <see langword="false" />.</value>
```